stories/RadioGroup/RadioGroup.regression-test.story.jsx

```diff
@@ -17,35 +17,34 @@ const options = [
   },
 ];
 
-const disabledOptions = [...options];
-disabledOptions[1] = {
-  value: 'Mustard',
-  label: 'Mustard',
-  disabled: true,
-};
+const disabledOptions = [
```

Comment:
Question. Have you noticed that this guy is adding another option and not just chaging the one it already have?

Reply:
Yep! It was already similar with this.